src/main/eo/org/eolang/txt/text.eo

```diff
@@ -69,3 +69,6 @@
 
   [i] > at
     ^.s.slice i 1 > @
+
+  # Returns
```

Comment:
@levBagryansky It would be better to use more detail comment, like `Make an integer out of this text`

Reply:
@levBagryansky What's up with this review comment?